.gitignore

```diff
@@ -6,3 +6,6 @@ dist
 .idea
 
 /.pnpm-store/
+.env
+
+tests/pnpm-lock.yaml
```

Comment:
we want to keep the lock file if any packages are added to test level in the future
```suggestion
```